node/src/BaseClient.ts

```diff
@@ -6526,8 +6528,13 @@ export class BaseClient {
      * console.log(newsChannels); // Output: ["news.sports", "news.weather"]
      * ```
      */
-    public async pubsubChannels(pattern?: string): Promise<string[]> {
-        return this.createWritePromise(createPubSubChannels(pattern));
+    public async pubsubChannels(
+        pattern?: GlideString,
+        decoder?: Decoder,
```

Comment:
Please use `DecoderOption`
```suggestion
        options?: DecoderOption,
```